src/include/gen/autoconfig_msvc.h

```diff
@@ -1,27 +1,18 @@
-/*
- * 2002.02.15 Sean Leyne - Code Cleanup, removed obsolete ports:
- *                          - MAC (MAC, MAC_AUX and "MAC_CP" defines)
- *                          - EPSON, DELTA, IMP, NCR3000, NeXT, M88K, Cray
- *                          - OS/2, Apollo
- *
- * 2002-02-23 Sean Leyne - Code Cleanup, removed old Win3.1 port (Windows_Only)
- *
- * 2002.10.27 Sean Leyne - Code Cleanup, removed obsolete "UNIXWARE" port
- * 2002.10.27 Sean Leyne - Code Cleanup, removed obsolete "Ultrix" port
- *
- * 2002.10.28 Sean Leyne - Completed removal of obsolete "DGUX" port
- * 2002.10.28 Sean Leyne - Code cleanup, removed obsolete "MPEXL" port
- * 2002.10.28 Sean Leyne - Code cleanup, removed obsolete "SGI" port
- *
- * 2002.10.29 Sean Leyne - Removed obsolete "Netware" port
- *
- * 2002.10.30 Sean Leyne - Removed support for obsolete "PC_PLATFORM" define
- *
- */
-
 #ifndef AUTOCONFIG_H
 #define AUTOCONFIG_H
 
+ // Windows 10 (0x0A00) is the baseline version for Firebird 6
+#define FB_WIN32_WINNT_BASELINE 0x0A00
+
+#ifndef _WIN32_WINNT
```

Comment:
Global defines, IMO, should be defined in property sheets, and not depend that everyone includes `firebird.h` first.

Reply:
Given the other configuration that autoconfig_msvc.h does, it already needs to be first, otherwise you run the chance that some of it is not actually defined (e.g. the `WIN32_LEAN_AND_MEAN` and `NOxxx` defines) before the Windows API is included.

Do you want me to move those also to, for example, Firebird.Common.props?

Reply:
OK, but should I define `_WIN32_WINNT` in Firebird.Common.props?

Reply:
> OK, but should I define `_WIN32_WINNT` in Firebird.Common.props?

I think - yes, but, please, add comment about it at autoconfig_msvc.h

Reply:
OK, done